frontend/src/metabase/components/InputBlurChange.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from "react";
+import _ from "underscore";
+import Input from "metabase/core/components/Input";
+// import { Input } from "./InputBlurChange.styled";
+
+/**
+ * A small wrapper around <input>, primarily should be used for the
+ * `onBlurChange` feature, otherwise you should use <input> directly
+ */
+
+interface InputBlurChangeProps {
```

Comment:
Wdyt about changing it like this?
```ts
import Input, { InputProps } from "metabase/core/components/Input";
// ...
interface InputBlurChangeProps extends InputProps {
  onBlurChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
}
```

Reply:
I had this thought earlier but clearly forgot to do it. Yes, much cleaner 👍